src/scipp/io/hdf5.py

```diff
@@ -148,8 +148,8 @@ def _data_handler_lut():
     from .._scipp.core import dtype as d
     handler = {}
     for dtype in [
-            d.float64, d.float32, d.int64, d.int32, d.bool, d.datetime64,
-            d.vector_3_float64, d.matrix_3_float64
+            d.float64, d.float32, d.int64, d.int32, d.bool, d.datetime64, d.vector3,
+            d.linear_transform3
```

Comment:
Can you open an issue to add support for writing and reading the new dtypes?

Reply:
Added at https://github.com/scipp/scipp/issues/2329